docs/api/field_types_reference/xmltextfield.md

```diff
@@ -0,0 +1,131 @@
+# XmlText Field Type
+
+The XmlText Field Type isn't officially supported by [[= product_name =]]. It can be installed by requiring `ezsystems/ezplatform-xmltext-fieldtype`. PlatformUI does not support WYSIWYG editing of Fields of this type.
```

Comment:
```suggestion
The XmlText Field Type isn't officially supported by [[= product_name =]]. It can be installed by requiring `ezsystems/ezplatform-xmltext-fieldtype`. The Back Office does not support WYSIWYG editing of Fields of this type.
```
PlatformUI is an outdated term for the Back Office in earlier product versions